linera-service/tests/end_to_end_tests.rs

```diff
@@ -853,6 +864,16 @@ impl Application {
         serde_json::from_value(response_body["accounts"].clone()).unwrap_or_default()
     }
 
+    async fn assert_fungible_app_has_accounts(
+        &self,
+        accounts: impl IntoIterator<Item = (AccountOwner, Amount)>,
```

Comment:
Actually the original terminology which was just "check_accounts" was more precise.
Here the name "has_accounts" means just that we check that the accounts are the ones present,
while actually we do more: We check the accounts and we check their amounts.

Reply:
Happy to replace with `assert_fungible_app_has_account_balances`. Check was really not saying much IMHO.